### ISSUE TITLE
Pressing Enter while adjusting round numbers should start scrambling

### ISSUE THREAD

**Issue by lgarron** [cubing/mark2#36]
Like an HTML for submits using Enter.

**Comment by MLSTRM** [comment]
I disagree, just because if you press Enter by accident during input, you have to start all over again.

**Comment by lgarron** [comment]
The URL hash updates. If you refresh the page, you should be exactly where you were.
